locale/zh-cn/download/releases.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: download-releases.hbs
+title: Previous Releases
+modules: "<code>NODE_MODULE_VERSION</code> refers to the ABI (application binary interface) version number of Node.js, used to determine which versions of Node.js compiled C++ add-on binaries can be loaded in to without needing to be re-compiled. It used to be stored as hex value in earlier versions, but is now represented as an integer."
+---
+
+### io.js &amp; Node.js
+Releases 1.x through 3.x were called "io.js" as they were part of the io.js fork. As of Node.js 4.0.0 the former release lines of io.js converged with Node.js 0.12.x into unified Node.js releases.
```

Comment:
nah, the pr title specifically says `(index/current)` so i assumed the other ones were going to be translated another time